front/src/components/Modal.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
prefer 
```suggestion
import type React from 'react';
```
(also for the other files please)